### PR TITLE
Fixture upload xlsx suffix

### PR DESCRIPTION
**Pull request by dannyroberts**
depends on https://github.com/dimagi/django-soil/pull/32
fixes http://manage.dimagi.com/default.asp?181058

### CODE REVIEW

File: corehq/apps/fixtures/views.py [hunk 1]
```diff
@@ -304,8 +304,9 @@ def get(self, request):
     def post(self, request):
         replace = 'replace' in request.POST
 
-        file_ref = expose_cached_download(request.file.read(),
-                                   expiry=1*60*60)
+        suffix = '.{}'.format(request.file.name.rsplit('.', 1)[-1])
+        file_ref = expose_cached_download(request.file.read(), expiry=1*60*60,
```
Comment (dimagimon) [inline corehq/apps/fixtures/views.py]: E226 missing whitespace around arithmetic operator

File: corehq/apps/locations/views.py [hunk 2]
```diff
@@ -642,8 +642,7 @@ def post(self, request, *args, **kwargs):
         domain = args[0]
 
         # stash this in soil to make it easier to pass to celery
-        file_ref = expose_cached_download(upload.read(),
-                                   expiry=1*60*60)
+        file_ref = expose_cached_download(upload.read(), expiry=1*60*60)
```
Comment (dimagimon) [inline corehq/apps/locations/views.py]: E226 missing whitespace around arithmetic operator

File: corehq/apps/products/views.py [hunk 3]
```diff
@@ -290,8 +290,7 @@ def post(self, request, *args, **kwargs):
 
         domain = args[0]
         # stash this in soil to make it easier to pass to celery
-        file_ref = expose_cached_download(upload.read(),
-                                   expiry=1*60*60)
+        file_ref = expose_cached_download(upload.read(), expiry=1*60*60)
```
Comment (dimagimon) [inline corehq/apps/products/views.py]: E226 missing whitespace around arithmetic operator

**Comment by dannyroberts** [comment]
@nickpell

**Comment by nickpell** [comment]
Looks great, waiting on tests

**Pull request merged**